rmk/Cargo.toml

```diff
@@ -41,10 +41,11 @@ num_enum = { version = "0.7", default-features = false }
 bitfield-struct = "0.8"
 byteorder = { version = "1", default-features = false }
 futures = { version = "0.3", default-features = false }
-sequential-storage = { version = "3.0.0", features = ["defmt-03"] }
+sequential-storage = { git="https://github.com/davidqge/sequential-storage.git", rev="edc04893e99ae38eaf62419298b3c72955d05840", features = ["defmt-03"] }
```

Comment:
I can merge this now, but using a git dependency will block the release to crates.io. So it's better to make it released to crates.io soon then we can switch to crates.io version.

Reply:
I removed last commit, so the other commits can be merged now.